pages/common/dcg.md

```diff
@@ -0,0 +1,16 @@
+# dcg
+
+> Drupal code generator.
+> More information: <https://github.com/Chi-teck/drupal-code-generator>.
+
+- Start a wizard to choose what kind a code (module, service, form, etc.) to generate:
+
+`dcg`
+
+- Specify directly the kind of code you want to generate:
```

Comment:
Generally we try to avoid the 2nd person ( i.e. `you`, `your`, etc) in tldr pages. What about this instead?
```suggestion
- Directly specify the kind of code to generate:
```